tests/std/tests/P0024R2_parallel_algorithms_transform_reduce/test.cpp

```diff
@@ -1,7 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-#pragma warning(disable : 4242 4244 4365) // test_case_incorrect_special_case_reasoning tests narrowing on purpose
+// test_case_incorrect_special_case_reasoning & test_case_narrowing_conversion tests narrowing on purpose

```

Comment:
Comment nitpicks: This should now say "test" to be grammatical. Additionally, we avoid abbreviating "and" to "&" in comments, to avoid confusion with its various meanings in C++.